modules/ichorcna/1.0/ichorcna.smk

```diff
@@ -0,0 +1,188 @@
+        plotYlim = CFG["options"]["run"]["ichorCNA_plotYlim"],
+        libdir = CFG["options"]["run"]["ichorCNA_libdir"]
+    conda: CFG["conda_envs"]["ichorcna"]
+    threads: CFG["threads"]["run"]
+    resources:
+        **CFG["resources"]["run"]
+    log:
+        stdout = CFG["logs"]["seg"] + "{seq_type}--{genome_build}/{binSize}/{tumour_id}--{normal_id}--{pair_status}.stdout.log",
+        stderr = CFG["logs"]["seg"] + "{seq_type}--{genome_build}/{binSize}/{tumour_id}--{normal_id}--{pair_status}.stderr.log"
+    shell:
+        "Rscript {params.rscript} --id {params.name} --libdir {params.libdir} --WIG {input.tum} --gcWig {params.gcwig} --mapWig {params.mapwig} --normalPanel {params.normalpanel} --ploidy \"{params.ploidy}\" --normal \"{params.normal}\" --maxCN {params.maxCN} --includeHOMD {params.includeHOMD} --chrs \"{params.chrs}\" --chrTrain \"{params.chrTrain}\" --genomeStyle {params.genomeStyle} --genomeBuild {params.genomeBuild} --estimateNormal {params.estimateNormal} --estimatePloidy {params.estimatePloidy} --estimateScPrevalence {params.estimateClonality} --scStates \"{params.scStates}\" --centromere {params.centromere} --exons.bed {params.exons} --txnE {params.txnE} --txnStrength {params.txnStrength} --minMapScore {params.minMapScore} --fracReadsInChrYForMale {params.fracReadsChrYMale} --maxFracGenomeSubclone {params.maxFracGenomeSubclone} --maxFracCNASubclone {params.maxFracCNASubclone} --plotFileType {params.plotFileType} --plotYLim \"{params.plotYlim}\" --outDir {params.outDir} > {log.stdout} 2> {log.stderr}"
+
+# Symlinks the final output files into the module results directory (under '99-outputs/')
+rule _ichorcna_output:
+    input:
+        corrDepth = str(rules._run_ichorcna.output.corrDepth),
+        param = str(rules._run_ichorcna.output.param),
+        cna = str(rules._run_ichorcna.output.cna),
+        segTxt = str(rules._run_ichorcna.output.segTxt),
+        seg = str(rules._run_ichorcna.output.seg),
+        plot = str(rules._run_ichorcna.output.plot)
+    output:
+        corrDepth = CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/corrDepth/{binSize}/{tumour_id}--{normal_id}--{pair_status}.corrDepth.txt",
+        param = CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/param/{binSize}/{tumour_id}--{normal_id}--{pair_status}.param.txt",
+        cna = CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/binCNA/{binSize}/{tumour_id}--{normal_id}--{pair_status}.cna.seg",
+        segTxt = CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/seg_txt/{binSize}/{tumour_id}--{normal_id}--{pair_status}.seg.txt",
+        seg = CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/seg/{binSize}/{tumour_id}--{normal_id}--{pair_status}.seg",
+        plot = CFG["dirs"]["outputs"] + "{seq_type}--{genome_build}/plot/{binSize}/{tumour_id}--{normal_id}--{pair_status}_genomeWide.pdf"
+    run:
+        op.relative_symlink(input.corrDepth, output.corrDepth)
```

Comment:
the latest version of oncopipe also supports argument `in_module=TRUE`, which creates "shallow" symlinks. You can add it here as well, and the recent modules have an example of this

Reply:
Added for all output relative_symlinks